config/config.go

```diff
@@ -262,6 +262,7 @@ type Config struct {
 	LogLevel                          string                                `json:"log_level"`
 	Security                          SecurityConfig                        `json:"security"`
 	EnableKeyLogging                  bool                                  `json:"enable_key_logging"`
+	DefaultVersionHeader              bool                                  `json:"default_version_header"`
```

Comment:
Since it is not `Default` anymore, and probably make sense rename option to just `VersionHeader`